src/utils/config.ts

```diff
@@ -5,6 +5,7 @@ export function buildConfig(params: CrossmintEmbedParams): CrossmintEmbedConfig
     const ret: CrossmintEmbedConfig = {
         libVersion: LIB_VERSION,
         apiKey: params.apiKey,
+        projectId: params.projectId ?? "crossmint",
```

Comment:
let's default to `undefined` being `"crossmint"` rather than that being something we explicitly set

Reply:
ah what i mean is: omit the `?? "crossmint"` and later, we'll interpret this value being missing / undefined as "backend, look up crossmint wallets"